packages/signers/src/turnkey/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { TStamper } from "@turnkey/http/dist/base";
+import type { Transport } from "viem";
+
+export interface TurnkeyAuthParams {
+  organizationId: string;
+  signWith: string;
```

Comment:
is this an `Address`?

Reply:
an `Address` or `ID`. i also describe it [here](https://github.com/alchemyplatform/aa-sdk/pull/292/files#diff-fdf51fa5129147a3ef7bf55922343105cd7e6f7d4995e147c019e81662794ca7R67) but lmk if some additional comments anywhere would be clarifying.